Sources/Contour/Caching/CacheMessageExchangeFilter.cs

```diff
@@ -1,82 +0,0 @@
-﻿using System.Threading.Tasks;
-
-using Contour.Filters;
-using Contour.Helpers;
-
-namespace Contour.Caching
-{
-    /// <summary>
-    /// Фильтр, который кеширует ответные сообщения.
-    /// </summary>
-    public class CacheMessageExchangeFilter : IMessageExchangeFilter
```

Comment:
Breaking backward compatibility. You should avoid it.

Reply:
Reverted deletion and marked as obsolete.